### PR TITLE
Several improvements and fixes

### PR DESCRIPTION
**Pull request by feckert**
- luci-app-openvpn: add missing license tag
- luci-app-firewall: remove proto "other" on porforwarding to fix luci exception
- luci-mod-admin-full: fix factory reset detection
- luci-mod-admin-full: remove led page if "/sys/class/leds" directory is empty

### CODE REVIEW

**Comment by feckert** [comment]
@hnyman the same on this PR.
Any progress or opinion?

**Comment by feckert** [comment]
@hnyman sorry for the additional ping but its now 1,5 month since last ping add 3,5 month since PR open. I know you have a lot todo as all developers. My be anyone else could have a look on this commits.
I think the luci-mod-admin-full commits are worth to take a look :-)

**Comment by hnyman** [comment]
I have hoped that @jow- would look on those.
I am not sure about the reasoning on some of the stuff, so I have left this for more qualified people ;-)

File: applications/luci-app-firewall/luasrc/view/firewall/cbi_addforward.htm [hunk 1]
```diff
@@ -43,7 +43,6 @@
 					<option value="tcp udp">TCP+UDP</option>
 					<option value="tcp">TCP</option>
 					<option value="udp">UDP</option>
-					<option value="other"><%:Other...%></option>
```
[the remark this answers is not in the document]
Comment (feckert) [reply applications/luci-app-firewall/luasrc/view/firewall/cbi_addforward.htm]: I have removed this change from the PR.
To get the other issues fixed/merged.
I will open a follow up PR for this issue.

**Comment by yousong** [comment]
Committed and pushed with minor changes.  Thank you!

**Pull request closed without merging**